Unable to compile on ubuntu 18.04

Hello,

I am trying to build the RecastDemo on Ubuntu 18.04. However, I get the following error when I run make in Build/gmake:
`==== Building DebugUtils (debug) ====
==== Building Detour (debug) ====
DetourAlloc.cpp
cc1plus: error: -Werror=class-memaccess: no option -Wclass-memaccess
Detour.make:133: recipe for target 'obj/Debug/Detour/DetourAlloc.o' failed
make[1]: *** [obj/Debug/Detour/DetourAlloc.o] Error 1
Makefile:46: recipe for target 'Detour' failed
make: *** [Detour] Error 2`

How can solve this issue?

Thankyou in advance.

--- a/RecastDemo/premake5.lua
+++ b/RecastDemo/premake5.lua
@@ -76,7 +76,7 @@ project "Detour"
 	-- linux library cflags and libs
 	configuration { "linux", "gmake" }
 		buildoptions { 
-			"-Wno-error=class-memaccess"
+			"-Wno-class-memaccess"
 		}
 
 
@@ -156,7 +156,7 @@ project "RecastDemo"
 			"`pkg-config --cflags gl`",
 			"`pkg-config --cflags glu`",
 			"-Wno-ignored-qualifiers",
-			"-Wno-error=class-memaccess"
+			"-Wno-class-memaccess"
 
 		}
 		linkoptions { 

